### ISSUE TITLE
Problem on Login

### ISSUE THREAD

**Issue by paxpelus** [wolfeidau/cognito-vue-bootstrap#5]
On the specific line https://github.com/wolfeidau/cognito-vue-bootstrap/blob/master/src/cognito.js#L128 it seems like you are using UserPoolId instead of IdentityPoolId.

**Comment by wolfeidau** [comment]
Yeah I need to review the terminology, it isn't very consistent at the moment, way to many similar sounding things.

**Comment by alexaandru** [comment]
This explains it very nice: https://serverless-stack.com/chapters/cognito-user-pool-vs-identity-pool.html 

I don't think this is a bug, it works for me just fine. IdentityPoolId would've been for logging in with Facebook or Google (etc.) which does not seem to be the intent here.
